Makefile

```diff
@@ -54,7 +57,11 @@ test: clean-cov fmt vet $(GINKGO)
 ## install: Build and install kuadrantctl binary ($GOBIN or GOPATH/bin)
 .PHONY : install
 install: fmt vet
+ifneq ($(GIT_HASH),$(GIT_DIRTY))
+	GOBIN=$(PROJECT_PATH)/bin $(GO) install -ldflags "-X 'github.com/kuadrant/kuadrantctl/version.GitDirty=$(GIT_DIRTY)' -X 'github.com/kuadrant/kuadrantctl/version.GitHash=$(GIT_HASH)'" 
+else 
```

Comment:
I think you should always pass some build time variable. There is no "else"

Reply:
Sure - this probably relates to an above suggestion. :)